scripts/juju-force-upgrade/main.go

```diff
@@ -58,7 +58,7 @@ func getState() (*state.State, error) {
 	st, err := state.Open(state.OpenParams{
 		Clock:              clock.WallClock,
 		ControllerTag:      config.Controller(),
-		ControllerModelTag: config.Model(),
+		ControllerModelTag: names.NewModelTag(modelUUID), // XXX field misnomer
```

Comment:
really? we shouldn't be using ControllerModel tag?

Reply:
that's a mistake that I realised later on in the PR, just failed to clean up - thanks, will fix

I do think the OpenParams requires too much to be passed in though. We should be able to figure out the controller and controller model tags from the database. The caller should only have to pass in a model tag (to open a specific model's State) at most. There's no good reason to treat the controller model specially, AFAICT.